README.md

```diff
@@ -129,6 +129,19 @@ Just make sure port 8080 is accessible.
 
 Create a JSON-encoded array.  You must escape quote characters when using a command-line option.
 
+#### My options are not being considered
+
+Make sure you put the options before the main script argument.
+
+Right:
+```
+node-debug --hidden=test.js app
+```
+Wrong:
+```
+node-debug app --hidden=test.js
+```
+
```

Comment:
This change is not so important to add it here.
I don't like to extend `Troubleshooting` any time. This part is already too big and prevents users see useful `config` info.

We have
>
### How do I specify files to hide?
Create a JSON-encoded array. You must escape quote characters when using a command-line option.
```
$ node-inspector --hidden='["node_modules/framework"]'
```

I prefer to make small update in this part - add `node-debug` example,